app/javascript/common/constants.js

```diff
@@ -0,0 +1,8 @@
+export const MAX_LENGTH_NAMES = 24;
+export const MAX_LENGTH_DESCRIPTIONS = 128;
+
+export const nameHelp = `Maximum is ${MAX_LENGTH_NAMES} characters.`;
```

Comment:
Might need to annotate these to support translation efforts 🤔 

Checkout the use of `sprintf`

Reply:
ps: love LOVE that they're all located in one place 😍

Reply:
I'll look into that now, I was a little worried about that actually. It wouldn't be the end of the world to hard-code the numbers into the strings. I'll look at `sprintf` first.